front/pages/ideas/create.vue

```diff
@@ -301,6 +301,9 @@ export default {
       position: fixed;
       bottom: 0;
     }
+    @media (max-width: $screen-md-min) {
```

Comment:
there is a vuetify class for this. hidden-md-and-down hidden-md-and-up etc etc

Reply:
@danleyb2 could you make this edit?

Reply:
@aldarund @vedmant This style targets `trix-toolbar` it's a child componet of `components/TrixWrapper.vue` and so there is no way of adding vuetify classes directly